eng/targets/Helix.Common.props

```diff
@@ -28,6 +28,10 @@
     <!--  TODO: re-enable Debian.9.Arm64.Open and Ubuntu.1804.Arm64.Open    -->
   </ItemGroup>
 
+  <ItemGroup Condition="'$(RunOnOSX1014)' == 'true'">
```

Comment:
What sets `$(RunOnOSX1014)`?

Reply:
That's why I left a comment on a test 😄 https://github.com/aspnet/AspNetCore/pull/10824#pullrequestreview-245248769